Tests/Units/DependencyInjection/Compiler/AddSpreadCompilerPass.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Spy\TimelineBundle\Tests\Units\DependencyInjection\Compiler;
+
+use mageekguy\atoum;
+use Spy\TimelineBundle\DependencyInjection\Compiler\AddSpreadCompilerPass as TestedModel;
+
+class AddSpreadCompilerPass extends atoum\test
```

Comment:
Test suffix is missing

Reply:
actually i followed the convention of AddLocatorCompilerPass